Automattic/jetpack
# Gutenburg blocks: use BlockIcon in MediaPlaceholder for Simple payments

Fixes #https://github.com/Automattic/jetpack/issues/11843

Since [this PR](https://github.com/Automattic/jetpack/pull/12160) has already addressed:
- Tiled gallery
- Slideshow

#### Changes proposed in this Pull Request:
* Use `BlockIcon` component for custom icons in media-placeholders for Simple payments.

#### Testing instructions:
- Build extensions yarn build-extensions
- Insert Simple Payments block to a post
- Ensure no visual differences in placeholder's icon placement/spacing/size

#### Before
![before](https://user-images.githubusercontent.com/1758399/85302090-62526700-b4b1-11ea-880d-8c36692b6a36.png)

#### After
![Screenshot 2020-06-23 at 10 49 24 AM](https://user-images.githubusercontent.com/1758399/85376142-01be3b00-b540-11ea-9ca2-e575d721dacd.png)


#### Proposed changelog entry for your changes:
- Gutenburg blocks: use BlockIcon in MediaPlaceholder for Simple payments
